Code/Tools/ProjectManager/Source/Settings.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Settings.h>
+
+#include <AzCore/Settings/SettingsRegistryMergeUtils.h>
+#include <AzCore/IO/ByteContainerStream.h>
+#include <AzCore/Utils/Utils.h>
+
+namespace O3DE::ProjectManager
+{
+    Settings::Settings(bool saveToDisk)
+        : m_saveToDisk(saveToDisk)
+    {
+        m_settingsRegistry = AZ::SettingsRegistry::Get();
+
+        AZ_Assert(m_settingsRegistry, "Failed to create Settings");
+    }
+
+    bool Settings::IsInitialized()
+    {
+        // Settings intialized correctly if it successfuly got the SettingsRegistry
+        return m_settingsRegistry;
+    }
```

Comment:
I actually don't think we need this function because the Application also calls AZ::SettingsRegistry::Get() before this class is created, so the app would have quit before we got to this point if it SettingsRegistry wasn't available